config/initializers/devise.rb

```diff
@@ -1,13 +1,16 @@
 require 'saml_idp_constants'
+require_relative '../../app/mailers/concerns/mailable'
```

Comment:
Are both the autoload and require_relative needed, or can the latter be removed now that we added the autoload?